test/integration/routes_test.rb

```diff
@@ -116,5 +116,9 @@ class RoutesTest < ActionDispatch::IntegrationTest
   test "test get request for spectrums anonymous" do
     assert_routing({ path: '/spectrums/anonymous', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'anonymous'})
   end
+
+  test "test get request for spectrums clone_search" do
+    assert_routing({ path: '/spectrums/clone_search', method: 'get' }, {controller: 'spectrums', action: 'clone_search'})
```

Comment:
@aniruddhabagal The errors are failing because of this line. Can you change it to this? Thanks!
```suggestion
    assert_routing({ path: '/spectrums/clone_search', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'clone_search'})
```

Reply:
Did the changes. Hope it'll get resolved. :-)